TestInterNodeBlockSync_WithBenchmarkConsensusBlocks flaky fails

Running in flakiness (>100 tests) fails sometimes when the Benchmark algo gets to a block race or sync issue and doesn't agree on latest block

t.go:31: error 2019-08-15T11:05:42.620176Z consensus algo refused to validate block node=d573cb service=block-storage error=aborting shared state update due to inconsistency function=blockstorage.(*Service).notifyConsensusAlgos source=/Users/noam/work/src/github.com/orbs-network/orbs-network-go/services/blockstorage/validate_block.go:157 _test=acceptance _test-id=acc-TestInterNodeBlockSync_WithBenchmarkConsensusBlocks-1565867142-749-BENCHMARK_CONSENSUS 
        
this is the validation of the block failing in benchmarkconsensus/common.go line 37 -> setLastCommittedBlock

--- a/test/acceptance/node_sync_test.go
+++ b/test/acceptance/node_sync_test.go
@@ -24,10 +24,7 @@ func TestInterNodeBlockSync_WithBenchmarkConsensusBlocks(t *testing.T) {
 	newHarness().
 		WithConsensusAlgos(consensus.CONSENSUS_ALGO_TYPE_BENCHMARK_CONSENSUS).
 		//WithLogFilters(log.ExcludeEntryPoint("BenchmarkConsensus.Tick")).
-		AllowingErrors(
-			"leader failed to save block to storage",                 // (block already in storage, skipping) TODO(v1) investigate and explain, or fix and remove expected error
-			"all consensus \\d* algos refused to validate the block", //TODO(v1) investigate and explain, or fix and remove expected error
-		).
+		//AllowingErrors().
 		WithSetup(func(ctx context.Context, network *Network) {
 			var prevBlock *protocol.BlockPairContainer
 			for i := 1; i <= 10; i++ {

--- a/test/acceptance/service_sync_test.go
+++ b/test/acceptance/service_sync_test.go
@@ -32,10 +32,8 @@ func TestServiceBlockSync_TransactionPool(t *testing.T) {
 	newHarness().
 		WithInitialBlocks(blocks).
 		WithConsensusAlgos(consensus.CONSENSUS_ALGO_TYPE_BENCHMARK_CONSENSUS). // this test only runs with BenchmarkConsensus since we only create blocks compatible with that algo
-		AllowingErrors(
-			"leader failed to save block to storage",                 // (block already in storage, skipping) TODO(v1) investigate and explain, or fix and remove expected error
-			"all consensus \\d* algos refused to validate the block", //TODO(v1) investigate and explain, or fix and remove expected error
-		).Start(t, func(t testing.TB, ctx context.Context, network *Network) {
+		//AllowingErrors().
+		Start(t, func(t testing.TB, ctx context.Context, network *Network) {
 
 		topBlockHeight := blocks[len(blocks)-1].ResultsBlock.Header.BlockHeight()
 

